satpy/readers/seviri_l1b_native.py

```diff
@@ -373,9 +396,18 @@ def get_area_extent(self, dataset_id):
 
             # get actual navigation parameters from trailer data
             data15tr = self.trailer['15TRAILER']
-            HRV_bounds = data15tr['ImageProductionStats']['ActualL15CoverageHRV']
-
-            # lower window
+            HRV_bounds = data15tr['ImageProductionStats']['ActualL15CoverageHRV'].copy()
+            if self.fill_hrv:
+                HRV_bounds['UpperEastColumnActual'] = 1
+                HRV_bounds['UpperWestColumnActual'] = 11136
```

Comment:
the hard coded value 11136 should probably defined and imported from `seviri_base.py`. it has `HRV_NUM_COLUMNS` defined already, the same should be defined for `HRV_NUM_ROWS`

Reply:
fix this in `seviri_hrit`-reader in the same PR, too?

Reply:
This is now fixed, also for the hrit-reader. I also made sure to add the attributes that went lost in case of hrv padding in the hrit-reader.